src/main/java/com/zebrunner/carina/demo/gui/pages/myfitnesspal/MyFitnessPalLogInPage.java

```diff
@@ -0,0 +1,55 @@
+package com.zebrunner.carina.demo.gui.pages.myfitnesspal;
+
+import com.zebrunner.carina.utils.R;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class MyFitnessPalLogInPage extends MyFitnessPalBasePage {
+
+    @FindBy(xpath = "//a/p[text()=\"Sign Up\"]")
+    ExtendedWebElement signUpBtn;
+
+    @FindBy(id = "email")
+    ExtendedWebElement emailInput;
+
+    @FindBy(id = "password")
+    ExtendedWebElement passwordInput;
+
+    @FindBy(xpath = "//span[text()=\"Log in\"]/..")
+    ExtendedWebElement logInBtn;
+
+
+    public MyFitnessPalLogInPage(WebDriver driver){
+        super(driver);
+    }
+
+    public MyFitnessPalSignUpPage goToSignUp(){
+        signUpBtn.click();
+        return new MyFitnessPalSignUpPage(getDriver());
+    }
+
+    public MyFitnessPalHomePageWithSession logIn(String email, String password){
```

Comment:
It's better this way
public MyFitnessPalHomePageWithSession logIn(User user) {
        setEmailInputField(user.getEmail())
        setPasswordInputField(user.getPassword())
        // clickOnLogInBtn();
        return new MyFitnessPalHomePageWithSession(getDriver());
    }